.github/workflows/wheels.yml

```diff
@@ -97,6 +101,13 @@ jobs:
             platform_id: manylinux_x86_64
             manylinux_image: manylinux2014
 
+          # On Python 3.11 only supports 64bit and is only available with manylinux2014
```

Comment:
Looking at the previous entry, is this correct?
```suggestion
          # NumPy on Python 3.11 only supports 64bit and is only available with manylinux2014
```

Reply:
I guess so... I can't remember why I modified the comment... :(